.editorconfig

```diff
@@ -0,0 +1,24 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.py]
+indent_style = space
```

Comment:
`indent_style` could be in `[*]` section as it is common for all files except `[Makefile]`

Reply:
Agree, but this PR already merged. You can send new PR with this optimization